lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -80,9 +80,9 @@
   @include monospace;
   background-color: var(--nx-swatch-grey-95);
   border: 1px solid var(--nx-swatch-grey-90);
-  border-radius: var(--nx-border-radius);
+  border-radius: 4px;
```

Comment:
I'm seeing some mockup issues here.  For one thing, I'm realizing that there is no navigable mockup for plain `nx-code` in kaliedo.  I had to do a search for "code" to find it under the symbols, here: https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/eKOOqjm .  That said, there are aspects of that symbol which do not match the nx-code + nx-text-link mockup but presumably should, notably the border radius.

Reply:
Hmm I just noticed that the way Firefox calculates the height of inline elements  is different from Chrome. So the height might be different by around 0.5 to 1 px depending on the typeface. Getting the height matching with Sketch is going to be difficult unless we turn this into `inline-block`.
Additionally, outline-offset scales the border-radius so it's hard to match it with the mockup 🤔 . We could approximate it to get it visually close, I'm going to investigate a way around this.